woodwork/table_accessor.py

```diff
@@ -82,7 +82,7 @@ def init(self,
             column_metadata (dict[str -> dict[str -> json serializable]], optional): Dictionary mapping column names
                 to that column's metadata dictionary.
             use_standard_tags (bool, optional): If True, will add standard semantic tags to columns based
-                specified logical type for the column. Defaults to True.
+                on the specified logical type for the column. Defaults to True.
```

Comment:
Can we also change the `__init__` docstring in `schema.py` with this typo?

Reply:
done